src/spyglass/common/common_device.py

```diff
@@ -282,6 +285,30 @@ def insert_from_nwbfile(cls, nwbf):
                 }
                 cls.insert1(device_dict, skip_duplicates=True)
                 device_name_list.append(device_dict["camera_name"])
+        # Append devices from config file
+        if "CameraDevice" in config:
+            for conf in config["CameraDevice"]:
+                device_dict = dict()
```

Comment:
python's dynamic typing will do the declaration for you on the next line ... which could also be removed on 278 above
```suggestion
```